frontend/public/manifest.json

```diff
@@ -6,16 +6,6 @@
       "src": "favicon.ico",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
-    },
-    {
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
```

Comment:
Removed unused images
![image](https://github.com/user-attachments/assets/13d65f42-26a0-4236-9de3-9681682a8d75)

Reply:
Thank you 🙏